app/views/movies/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Movie</h1>
+
+<section>
+  
+  <p> Title: <%= @movie.title %> </p>
+  <p> Year: <%= @movie.creation_year %> </p>
+  <p> Genre: <%= @movie.genre %> </p>
+  <p> Actor by age:</p>
+  <% @movie.young_to_old_actors.each do |actor| %>
+    <%= actor.name %>
+  <% end %>
+  <p>Average Age: <%= @movie.average_age %> </p>
+
+  <h1>Add Actor</h1>
+<% @movie.actors.each do |actor| %>
+  <% if actor.id.present? %>
+<%= form_with url: "/movies/#{@movie.id}", method: :post, local: true do |form| %>
```

Comment:
Because you have this form nested inside `@movie.actors.each`, it will render a form for every actor that already belongs to a movie. Instead, I would get rid of lines 15 and 16 altogether and just render the form. We should be able to add an actor to a movie even if there aren't any actors assigned to a movie yet. This also would resolve the ambiguity error you were getting from Capybara. It was ambiguous because the form appeared multiple times so capybara didn't know which form it wanted it to fill it.

Reply:
I would also think about if `"/movies/#{@movie.id}"` is the path and controller we'd want to use here. I would expect the post/create action in the movie controller to create a new movie, but what we actually want is to create a record of an actor and movie's relationship in a join table.